cli/slsa-verifier/verify/options.go

```diff
@@ -145,27 +162,34 @@ var _ Interface = (*VerifyVSAOptions)(nil)
 func (o *VerifyVSAOptions) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringArrayVar(&o.SubjectDigests, "subject-digest", []string{},
 		"the digests to be verified. Pass multiple digests by repeating the flag. e.g. --subject-digest <digest type>:<digest value> --subject-digest <digest type>:<digest value>")
+	cmd.RegisterFlagCompletionFunc("subject-digest", cobra.NoFileCompletions)
```

Comment:
If we don't add this line, would the behavior be the same?

Reply:
No, it will suggest all filenames in the current dir as completions.